src/main/java/org/vufind/solr/indexing/PrintBrowseHeadings.java

```diff
@@ -0,0 +1,255 @@
+package org.vufind.solr.indexing;
+
+//
+// Author: Mark Triggs <mark@teaspoon-consulting.com>
+//
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.charset.Charset;
+
+// Note that this version is coming from Solr!
+import org.apache.commons.codec.binary.Base64;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.index.DirectoryReader;
+import org.apache.lucene.index.IndexNotFoundException;
+import org.apache.lucene.index.IndexReader;
+import org.apache.lucene.index.StoredFields;
+import org.apache.lucene.index.Term;
+import org.apache.lucene.search.ConstantScoreQuery;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.TermQuery;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.search.TotalHitCountCollector;
+import org.apache.lucene.store.FSDirectory;
+import org.vufind.util.BrowseEntry;
+import org.vufind.util.Utils;
+
+
+public class PrintBrowseHeadings
```

Comment:
It's odd that Git isn't recognizing this as a file rename, even though all the others got picked up correctly! Oh well.

Reply:
Yeah.  By default, git requires files at different paths to be 50% similar to detect renames, and `PrintBrowseHeadings` bore the brunt of the changes associated with the class renames and autoclose changes, so it's tipped over that threshold.

On the command line I can get it to pick up the rename by reducing that threshold to 20%:

```git diff --stat -M0.2 origin/dev VUFIND-1342```

but that doesn't help the web UI much :)